packages/components/ng-log-to-customer/src/component.js

```diff
@@ -8,7 +8,8 @@ export default {
     tile: '?rightTile',
   },
   bindings: {
-    source: '<',
+    logEndpoint: '<',
+    kindEndpoint: '<',
```

Comment:
names are too generic
```suggestion
    logApiUrl: '<',
    logKindApiUrl: '<',
```